doc/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ For F3D users:
  - Fixed dependency issue with the headless .deb binary release
  - Fixed a crash when using a file with more than 254 bones (Requires OpenGL 4.3)
  - Fixed an issue with Unicode filenames on the title bar on Windows
+ - Fixed an issue where focal point picking would pick the grid
```

Comment:
Maybe something more like
```suggestion
 - Fixed an issue where focal point picking would generate error messages
```
not to make it sound like the grid was actually getting picked ? (which it wasn't and the user-facing behavior hasn't changed, it's just quiet now)

Reply:
right, I should have read more carefully :)